lib/literature/router.ex

```diff
@@ -147,11 +147,17 @@ defmodule Literature.Router do
         end
       end
   """
+
+  # credo:disable-for-next-line Credo.Check.Refactor.CyclomaticComplexity
```

Comment:
@rcmonsayac why do we need to disable Credo shouldnt we refactor ?

Reply:
@dgross881 tried refactoring to reduce if statements inside `literature/2` macro